src/test/java/camp/nextstep/edu/calculator/CalculatorTest.java

```diff
@@ -0,0 +1,71 @@
+package camp.nextstep.edu.calculator;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.springframework.beans.factory.annotation.Value;
+
+class CalculatorTest {
+
+    Calculator calculator;
+
+    @BeforeEach
+    void setUp() {
+        calculator = new Calculator();
+    }
+
+    @DisplayName("빈 문자열이 입력될 경우 0을 반환한다")
+    @Test
```

Comment:
`@NullSource`와 `@EmptySource`를 사용하면 어떨까요? 아래의 글을 참고하세요.
https://www.baeldung.com/parameterized-tests-junit-5

```suggestion
    @ParameterizedTest
    @NullSource
    @EmptySource
```